docs/source/api/plugin/subscription-callback.mdx

```diff
@@ -0,0 +1,121 @@
+---
+title: "API Reference: Subscription callback plugin"
+api_reference: true
+minVersion: 4.9.0
+---
+
+> This document covers the usage of the Subscription callback plugin for use in Apollo Federation with Apollo Router. For more information about the protocol itself, see the [subscription callback protocol](/router/executing-operations/subscription-callback-protocol).
```

Comment:
This is getting wrapped so it looks a little funky:
<img width="761" alt="image" src="https://github.com/apollographql/apollo-server/assets/23219998/7a3e8638-b05b-49b2-9970-51edd774b79e">


Might be easier to just wrap the whole thing in a `<blockquote>`